suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -27,6 +27,11 @@ const transactionIconMap: Partial<Record<TransactionType, IconName>> = {
     sent: 'send',
 };
 
+const transactionTextMap: Partial<Record<TransactionType, 'Sent' | 'Received'>> = {
```

Comment:
`Partial<Record<TransactionType, string>>` is correct type, oterwise you will have to change type everytime when you change value

Reply:
https://github.com/trezor/trezor-suite/pull/7272/commits/39ce90eab3ceee16f586a865709b88eb24350166